test/e2e/pod_rm_test.go

```diff
@@ -189,11 +192,19 @@ var _ = Describe("Podman pod rm", func() {
 
 		session = podmanTest.Podman([]string{"pod", "rm", "bogus", "test1"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(1))
+		expect := "no pod with name or ID bogus found: no such pod"
+		if IsRemote() {
+			expect = `unable to find pod "bogus": no such pod`
+		}
+		Expect(session).Should(ExitWithError(1, expect))
 
 		session = podmanTest.Podman([]string{"pod", "rm", "test1", "bogus"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(1))
+		// FIXME: WEIRD! Two different failure messages:
+		//    no pod with name or ID test1 found: no such pod
+		//    test1 is a container, not a pod: no such pod
+		// no clue why (it's not remote vs local, nor fedora/debian)
```

Comment:
Another weird one.

Reply:
sqlite vs boltdb

Reply:
vs remote. Thanks for the boltdb tip, this is now "fixed" (in the sense of, hideous conditionals but better coverage)